src/khoj/routers/api.py

```diff
@@ -862,3 +862,7 @@ async def extract_references_and_questions(
         compiled_references = [item.additional["compiled"] for item in result_list]
 
     return compiled_references, inferred_queries, defiltered_query
+
+@app.get("/health")
```

Comment:
Ah, my bad for not realizing this before, but this should be as such
```suggestion
@api.get("/health")
```

You can run the unit tests locally by enabling your virtual environment with the development packages and then running `pytest` from the cli.